site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -161,6 +161,9 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#{base_field_name}.externalIdentifierConfiguration.emailVerificationIdTimeToLiveInSeconds# [type]#[Integer]# [since]#Available since 1.8.0#::
 The time in seconds until an email verification Id is no longer valid and cannot be used by the Verify Email API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator# [type]#[Integer]# [optional]#Optional# [since]#Available since 1.27.0#::
```

Comment:
When we document values in the response, we don't add optional or required.

Reply:
Makes sense!